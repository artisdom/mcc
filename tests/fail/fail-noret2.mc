int main()
{
  if (true) return 0;
  else print(1);
}
