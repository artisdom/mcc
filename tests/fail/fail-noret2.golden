Type error: expected one of [int] but got void. Error occured in statement:
{
    if (true) return 0;
    else printf("%d ", 1);
}